app/1.0/docs/tools/polymer-cli.md

```diff
@@ -296,36 +296,37 @@ If all of the elements you want to test are in the same directory, you can speci
 
 Generates a production-ready build of your app. This process includes minifying the HTML, CSS, and JS of the application dependencies, and generating a service worker to pre-cache dependencies.
 
-Polymer CLI's build process is designed for apps that follow the [app shell architecture](https://developers.google.com/web/updates/2015/11/app-shell).
-The build command is typically run with the following options:
-
-*   `entrypoint`. The main HTML file that is imported for all routes. This file
-    should import the app shell file specified in the `shell` option. It should
-    be minimal since it's loaded and cached for each route.
-*   `shell`. The app shell file containing common code for the app.
-*   `fragment`. Any imports that are not synchronously loaded from the
-    app shell, such as async imports or any imports loaded on-demand (e.g. by
-    `importHref`). Can be a space-separated list of values, or the option
-    can be repeated.
-*   `sw-precache-config`. The underlying library that Polymer CLI uses to
-    generate service workers, `sw-precache`, accepts various configurtion
-    options. You can specify a path to a `sw-precache` config file with
-    this option. See [Options parameter][op] from the `sw-precache` README
-    for more information.
-
-For example, in a newly created `basic` app project you would run the following command to generate a build:
+Polymer CLI's build process is designed for apps that follow the [app shell architecture](https://developers.google.com/web/updates/2015/11/app-shell). To make sure your app builds properly, create a `polymer.json` file at the top-level of your project and store your
+build configurations there. The following properties can be used:
+
+- `entrypoint`: The main HTML file that is imported for all routes. This file should import the app shell file specified in the shell option. It should be minimal since it's loaded and cached for each route.
+- `shell`: The app shell file containing common code for the app.
+- `fragment`: Any imports that are not synchronously loaded from the app shell, such as async imports or any imports loaded on-demand (e.g. by importHref). Can be a space-separated list of values, or the option can be repeated.
+- `sources`: An optional list of globs matching your application source files. This will default to all files in your project `src/` directory, but configuring your own sources can be useful when you have other files that need to be included.
+- `includeDependencies`: An optional list of globs matching any additional dependencies you'd like to include with your build. If you end up with missing files that weren't detected in your application you can include them here to make sure that they are always added to your build.
```

Comment:
The difference between `sources` and `includeDependencies` still doesn't seem clear. My immediate thought is "why not just put everything in sources?" You might explain that the two are treated differently by the polymer-build task. Also, it might be good to reconcile the names, e.g. `includeSources`, `includeDependencies` or `sources`, `dependencies`